### DISCUSSION TITLE
Youtube

### DISCUSSION THREAD

**Discussion by JSMDC** [Syncplay/syncplay#691] [Q&A]
I want to try playing youtube url in syncplay but this error occur. What should I do? 
![image](https://github.com/user-attachments/assets/4f5d9bb1-4f99-4867-b204-c279c43b8244)

**Comment by Et0h** [comment]
> I want to try playing youtube url in syncplay but this error occur. What should I do? 

The first thing to check is if the same error occurs when you are not using Syncplay, e.g. if you are just opening the URL in VLC. If so, it's a VLC issue rather than a Syncplay issue. It might be fixed in an update to VLC if you're using an older build.

My general recommendation is that people use mpv (or mpv.net) with Syncplay as it generally has the best compatibility with our software. If you want to stream media on mpv/mpv.net you will probably need to also install yt-dlp.

**Comment by Qwerty-Space** [comment]
I would suggest using [mpv](https://mpv.io/) or [mpv.net](https://github.com/mpvnet-player/mpv.net) with an up to date version of [yt-dlp](https://github.com/yt-dlp/yt-dlp)
